components/layout_2020/query.rs

```diff
@@ -392,6 +394,26 @@ pub fn process_resolved_style_request_for_unstyled_node<'dom>(
     style.computed_value_to_string(PropertyDeclarationId::Longhand(longhand_id))
 }
 
+fn shorthand_to_css_string(
+    id: style::properties::ShorthandId,
+    style: &style::properties::ComputedValues,
+) -> String {
+    use style::values::resolved::Context;
+    let mut block = PropertyDeclarationBlock::new();
+    let mut dest = String::new();
+    for longhands in id.longhands() {
+        block.push(
+            style.computed_or_resolved_declaration(longhands, Some(&Context { style })),
+            Importance::Normal,
+        );
+    }
+    let css = block.shorthand_to_css(id, &mut dest);
+    match css {
+        Ok(_) => return dest.to_owned(),
+        Err(_) => return String::new(),
+    }
```

Comment:
You can omit `return`:

```suggestion
    match block.shorthand_to_css(id, &mut dest) {
        Ok(_) => dest.to_owned(),
        Err(_) => String::new(),
    }
```